src/cmd/cmd.zig

```diff
@@ -11,6 +11,7 @@ const vm_core = @import("../vm/core.zig");
 const RunContext = @import("../vm/run_context.zig").RunContext;
 const relocatable = @import("../vm/memory/relocatable.zig");
 const Config = @import("../vm/config.zig").Config;
+const build_options = @import("../main.zig").build_options;
```

Comment:
I would avoid using things from main because the VM can be used as a library and not always as an executable.

Reply:
oooh I just saw that there was some constant options there and thought it would be the right place.

Where should I put this then? A separate "build_options.zig" file?